apps/clinical/src/components/forms/observations/__tests__/ObservationFormsEndToEnd.integration.test.tsx

```
@@ -0,0 +1,1082 @@
+// ──────────────────────────────────────────────────────────────────────────────
+
+describe('5. Validation prevents invalid data from reaching the bundle', () => {
+  it('should NOT add observations to store when validation error occurs', async () => {
+    setupMetadataMock();
+    // Empty form — no observations
+    mockGetValue.mockReturnValue({ observations: [], errors: [] });
+
+    const store = useObservationFormsStore.getState();
+    store.addForm(mockVitalsForm);
+
+    const onFormObservationsChange = jest.fn((
```

ESLint

[error] Delete `(`

(prettier/prettier)